Gopkg.toml

```diff
@@ -42,25 +42,25 @@ required = [
 
 [[constraint]]
   name = "github.com/golang/mock"
-  version = "1.2.0"
+  branch = "master"
```

Comment:
can we point to `v1.2.0` ?

Reply:
I'm mimicking the capa repo here.

Reply:
But pointing to `master` is not advisable. I would encourage even the AWS provider to follow suit.

Reply:
Fixed.